.pre-commit-hooks.yaml

```diff
@@ -16,7 +16,7 @@
     name: clang-format-diff
     entry: clang_format.py
     language: script
-    args: ['--diff']
+    args: ['11.0.0','diff']
 
     # git-clang-format does not take kindly to being run in parallel; it writes
     # to a speicfic file in .git.
```

Comment:
Update this comment / move it to where it's needed now?

Reply:
The comment at the same location earlier as i haven't done any changes to git-clang-format file I haven't updated this. Let me know if I misunderstood.